packages/core/components/Drawer/styles.module.css

```diff
@@ -1,26 +1,47 @@
+.Drawer {
+  font-family: var(--puck-font-family);
+}
+
 .DrawerItem:last-of-type .DrawerItem-default .DrawerItem-draggableWrapper {
-  padding-bottom: 0px;
+  margin-bottom: 0;
```

Comment:
Does this need to go back to `padding` following https://github.com/measuredco/puck/pull/318#discussion_r1445441893?

Reply:
Fixed